RockWeb/Blocks/Groups/GroupMemberAddFromUrl.ascx.cs

```diff
@@ -118,6 +129,20 @@ protected override void OnLoad( EventArgs e )
                     return;
                 }
 
+                // Validate the group type.
+                if ( !string.IsNullOrEmpty( GetAttributeValue( "LimitGroupType" ) ) )
+                {
+                    bool groupTypeMatch = GetAttributeValue( "LimitGroupType" )
+                        .Split( ',' )
+                        .Contains( group.GroupType.Guid.ToString(), StringComparer.CurrentCultureIgnoreCase );
+
```

Comment:
@cabal95 This request looks good. Thanks! In future you could use something like this here: 
`GetAttributeValue( "LimitGroupType" ).SplitDelimitedValues().AsGuidList().Contains( group.GroupType.Guid );`